app/Http/Transformers/v3SignupTransformer.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace Rogue\Http\Transformers;
+
+use Rogue\Models\Signup;
+use Rogue\Services\Registrar;
+use League\Fractal\TransformerAbstract;
+
+class v3SignupTransformer extends TransformerAbstract
```

Comment:
@DFurnes is this name ok? I'm horrible at naming things! 🙈

Reply:
Let's put this in a `Rogue\Http\Transformers\Three` namespace like the controller!